Assets/_BForBoss/_Entities/Scripts/LifeCycleBehaviour.cs

```diff
@@ -17,6 +18,11 @@ public abstract class LifeCycleBehaviour : MonoBehaviour
 
         public bool IsAlive => _lifeCycle.IsAlive;
 
+        public void AddLifeCycleSubscriber(LifeCycleManager manager)
+        {
+            _lifeCycle.OnDeath += manager.LivingEntitiesAmountChanged;
```

Comment:
I think it's better to put this subscription within the `LifeCycle` class instead.

Reply:
And also food for thought. 

Should each Living Entity need to know about the existance of a manger? or should it be the other way around?

Reply:
Yeah I think currently the lifecycle doesn't know about the manager and I think we should aim to keep it that way, in my opinion

Reply:
Fair, will make the change.